src/modules/SX127x.cpp

```diff
@@ -327,7 +363,31 @@ int16_t SX127x::receive(uint8_t* data, size_t len) {
       delete[] data;
       data = new uint8_t[length + 1];
     }
-    _mod->SPIreadRegisterBurst(SX127X_REG_FIFO, length, data);
+    else{
```

Comment:
Why is the actual reception code in else branch? It won't be able to receive anything if user calls it with String class (len parameter equal to zero).